CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+# Contributing to OurPromise 🤝🏻
+
+🎊If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🎊
+
+This project is actively looking for contributors!
+The following is a set of guidelines for contributing to this project, and you can see the live instance of the project [here](https://ourpromise.web.app/).
+
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+**Table Of Contents**
+
+[Code of Conduct 🗒](#code-of-conduct-🗒)
+
+[Before You Contribute 🏁](#before-you-contribute-🏁)
+
+[Local Setup 💻](#local-setup-💻)
+
+[How Can You Contribute 📝](#how-can-you-contribute-📝)
+
+- [Reporting Bugs 🐛](#reporting-bugs-🐛)
+- [Suggesting Enhancements ✨](#suggesting-enhancements-✨)
+- [First Code Contribution 🥇](#first-code-contribution-🥇)
+- [Pull Request 🥳](#pull-request-🥳)
+
+[Getting Your Changes Reviewed 📝](#getting-your-changes-reviewed-📝)
+
+[Pull Request Reviewers Guide 🦮](#pull-request-reviewers-guide-🦮)
+
+## Code of Conduct 🗒
+
+Everyone participating in this project is requested to practice decent behaviour. Help others, share knowledge and grow your knowledge!
```

Comment:
[Note]
this is good for now, but we will need `CODE_OF_CONDUCT.md` to specifically cover this topic. 
The issue opened at #49

Reply:
Should I add these files (leave them empty) and link them through contributing.md file?

Reply:
@kachun333 What to do?

Reply:
@aswin2108 , sorry I missed your comment.

We can just leave it as it is, no action required for now